ArchiSteamFarm/Localization/Strings.resx

```diff
@@ -699,4 +699,10 @@ Process uptime: {1}</value>
 		<value>{0} config file will be migrated to the latest syntax...</value>
 		<comment>{0} will be replaced with the relative path to the affected config file</comment>
 	</data>
+	<data name="WarningWeakIPCPassword" xml:space="preserve">
+		<value>Your IPC password is weak. Please consider choosing a stronger one! You can find guidelines on strong passwords in our wiki!</value>
```

Comment:
```suggestion
		<value>Your IPC password seems to be weak. Consider choosing a stronger one for increasing your security.</value>
```

Reply:
Maybe "increased security" instead of "increasing security"?

Reply:
Sounds better indeed, go ahead.

Reply:
The main purpose is to correct that we >believe< the password is weak, not claim that it really is. Also I removed note about the wiki because I don't believe we need to educate users on strong passwords, it should be common knowledge and if it isn't, then google is an option.